### PR TITLE
StationX 

### PR DESCRIPTION
**Pull request by toshitsom**
# Pull request checklist

Use the following template to make sure your PR fits the ConsenSys documentation standard.

## Before creating the PR

Make sure that:

- [ ] You've read the [contribution guidelines](https://docs-template.consensys.net/getting-started/style-guide).
- [ ] You've previewed your changes locally.

## Describe the change

Updated the links that went to blockscout and changed them to the goerli.lineascan.build

## Issue fixed

No current issue exists

## Impacted parts

For content changes:

- [ x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've fixed any issues raised by the tests.
- [ ] You've previewed your changes on Vercel below.

### CODE REVIEW

File: docs/use-linea/explore/use-stationx.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+﻿# <a name="_q4do3bv07v1f"></a>Explore StationX on Linea Testnet
```
Comment (emilyJLin95) [inline docs/use-linea/explore/use-stationx.md:1]: Start eh file with
```
---
title: Use StationX
sidebar_position: 1
---
```

Any read you need to add this `<a` tag

File: docs/use-linea/explore/use-stationx.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+﻿# <a name="_q4do3bv07v1f"></a>Explore StationX on Linea Testnet
+StationX is a protocol that automates workflows using account abstraction for communities to manage money towards shared goals. The platform lets anyone spin up a Station in less than 60 seconds to distribute membership tokens/NFTs to contributors, manage assets together in a treasury & governance on a single dashboard. You can read more about it [here](https://stationx.network)
+
+Before we get started, ensure your wallet is [configured to use Linea](https://docs.linea.build/use-linea/set-up-your-wallet). You need to have sufficient lineaETH (0.1 lineaETH recommended) to complete all steps, get lineaETH from [here](https://faucet.goerli.linea.build/). 
+
+
+
+<h2> Step 1: Create a Station </h2>
```
Comment (emilyJLin95) [inline docs/use-linea/explore/use-stationx.md:8]: Should use `##`

File: docs/use-linea/explore/use-stationx.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- Click 'Next' to set Governance (Coming Soon), currently you can only create Stations with governance **disabled** on Linea.  
+<img width="720" alt="Screenshot 2023-06-11 at 12 36 02" src="https://github.com/toshitsom/doc.zk-evm/assets/22795668/b31d3dc7-9627-4afe-ab6e-da481bc513ab">
+
+- Click **‘Finish’**, approve the transaction in your wallet & your **Station is live.** 
+<img width="720" alt="Screenshot 2023-06-13 at 17 52 22" src="https://github.com/toshitsom/doc.zk-evm/assets/22795668/5d05aaa5-7183-4834-b2c6-4b43026d4bcf">
+
+- You're done with Step 1, let's move to Step 2!
+
+
+<h2>Step 2: Claim 100 USDC </h2>
```
Comment (emilyJLin95) [inline docs/use-linea/explore/use-stationx.md:35]: `##`

File: docs/use-linea/explore/use-stationx.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+- Add token address - **0x45a27ea11D159A86AacE1eC24d3ba3d103642D9f**
+
+<img width="350" alt="Screenshot 2023-06-11 at 12 43 10" src="https://github.com/toshitsom/doc.zk-evm/assets/22795668/2428e33c-ea1e-4f50-a3e1-f1bdab6e2665">
+
+- You’ll be able to see the balance in your wallet
+
+- You're done with Step 2, let's move to Step 3!
+
+
+<h2>Step 3: Mint “Explorer” NFT by StationX</h2>
```
Comment (emilyJLin95) [inline docs/use-linea/explore/use-stationx.md:54]: `##`

**Comment by subham7** [comment]
@emilyJLin95 I have fixed all the syntax issues please could you review it

File: project-words.txt [hunk 5]
```diff
@@ -681,6 +681,8 @@ Postbots
 permissionlessly
 redcircle
 bluecircle
+Degen
```
Comment (emilyJLin95) [inline project-words.txt:684]: can you put these in alphabetical order? I know that none of the others are at the bottom, but if you can put it towards the top where it is alphabetical, that would be awesome @toshitsom

Comment (subham7) [reply project-words.txt]: I have arranged the works please check

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Pull request merged**